lib/manifest.js

```diff
@@ -0,0 +1,78 @@
+let path = require("path");
+let { createFile, generateFingerprint, uriJoin } = require("faucet-pipeline-util");
+let retry = require("./retry");
+let retryTimings = [10, 50, 100, 250, 500, 1000];
+
+class Manifest {
+	constructor(config, fingerprint, configDir, exitOnError) {
+		this.config = config;
+		this.manifest = {};
+		this.fingerprint = fingerprint;
+		this.configDir = configDir;
+		this.exitOnError = exitOnError;
+	}
+
+	add(targetPath, content, opts = {}) {
+		let error = opts.error;
+		let outputPath = targetPath;
+
```

Comment:
FWIW, I'd shift this blank line upwards by one in order to group `outputPath` definition statements.